src/scss/components/_view_instant.scss

```diff
@@ -1,4 +1,4 @@
-@import '../vars';
+@import '../abstracts/variables';
```

Comment:
Why do we need to import this here? We also removed the `@import './vars';` from other files.